game/Content/Globals.yaml

```diff
@@ -52,6 +52,7 @@ Globals:
   DisableRemnantStory: false
   DisablePirates: false
   AIUsesPlayerDesigns: true
+  EnableRandomizedAIFleetSizes: true
```

Comment:
Add here  a comment explaining what it does. Also add it to example mod global.yaml , maybe.